build.gradle

```diff
@@ -840,10 +840,15 @@ task cleanAll {
 }
 task buildAll {
   dependsOn getTasksByName('assemble', true).collect { it.path }
+  dependsOn getTasksByName('product', true).collect { it.path }
   dependsOn getTasksByName('testClasses', true).collect { it.path }
   dependsOn distTar, distZip
   mustRunAfter cleanAll
 }
+task buildDtests {
+  delete "${snappyProductDir}/dtests/build-artifacts/scala-2.11/"
```

Comment:
path can be changed as 
 "${snappyProductDir}/dtests/build-artifacts/scala-${scalaBinaryVersion}/"